.github/workflows/spell_check.yml

```
@@ -0,0 +1,14 @@
+name: Test GitHub Action
+on: [pull_request]
```

```suggestion
on:
  pull_request:
    types: [opened, synchronize]
```
Otherwise, I don't think it would rerun after updates.